grow/pods/pods.py

```diff
@@ -144,8 +149,8 @@ def _get_bytecode_cache(self):
     def _load_experiments(self):
         config = self.yaml.get('experiments', {})
         for key, value in config.iteritems():
-            # Experiments can be turned on with a True value.
-            # But the true does does not act as a configuration.
+            # Features can be turned on with a True value
```

Comment:
I'm going to a separate pull request to clean up this process of reading from a config into the features object so it doesn't need to be part of the pod.